### DISCUSSION TITLE
HC_envSet/CTFtool/

### DISCUSSION THREAD

**Discussion by giscus** [ProbiusOfficial/Hello-CTF#54] [Show and tell]
# HC_envSet/CTFtool/

开源免费、新手友好的CTF(Capture The Flag,夺旗赛)入门教程

https://ctf.probius.xyz/HC_envSet/CTFtool/

**Comment by Strawberry47** [comment]
Clash的github链接打不开啦

**Reply by Randark-JMT** [reply]
已更新为 clash_singbox-tools 的项目地址
Commit: https://github.com/ProbiusOfficial/Hello-CTF/commit/9b9c8e15e9bdaf218358f4e75c0f576364d49288

**Comment by 3699999** [comment]
Clash-verge更好用一点嘛

**Comment by lemonllm** [comment]
web环境设置和MISC环境配置 404了

**Comment by asdcvxsda** [comment]
什么是时候更新完成啊 在网上下的 linux命令lsb wbm ... 缺包还有权限不够
